roles/ceph-common/tasks/installs/debian_ceph_repository.yml

```diff
@@ -46,7 +46,11 @@
 
 - name: add gluster nfs ganesha repo
   apt_repository:
-    repo: "ppa:gluster/nfs-ganesha"
+    repo: "{{ item }}"
     state: present
+  with_items:
+    - ppa:gluster/libntirpc
```

Comment:
same as below, jemalloc is needed by osds not nfs, if nfs is not enabled this repo won't get added